srcpkgs/gns3-gui/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'gns3-gui'
 pkgname=gns3-gui
-version=2.2.10
-revision=2
+version=2.2.17
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-psutil python3-jsonschema python3-sentry
  python3-PyQt5-svg python3-PyQt5-websockets python3-sip xterm inetutils-telnet
  gns3-net-converter python3-distro"
 checkdepends="${depends}"
 short_desc="Graphical Network Simulator 3 - GUI"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
+maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gns3.com"
 distfiles="https://github.com/GNS3/${pkgname}/archive/v${version}.tar.gz"
-checksum=0f04fe1b3b328bab5ee80755ea90e18e0d25828fbf4a644f3fbec29189f19bc8
+checksum=@2200216046969000e6f076707d2ea3938a0777dc82e3d940cc41dc26c7fd46b5
 
 pre_build() {
```

Comment:
Please change `pre_build` to `post_patch`. `pre_build` only runs during the build stage, but sometimes it is nice to verify that the patching had the intended effect by running `xbps-src patch` without triggering the build stage.